examples/internal/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -386,6 +388,11 @@ message UpdateV2Request {
 //
 // See: https://google.aip.dev/123
 message Book {
+	option (google.api.resource) = {
+		type: "library.googleapis.com/Book"
+		pattern: "publishers/{publisher}/books/{book}"
+	};
```

Comment:
This was to match the [AIP for Resource Types](https://google.aip.dev/123#annotating-resource-types), similar with the other comment removed since they do not change behavior as you said